packages/imports-parser/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@tokey/imports-parser",
+  "description": "parser for esm like imports",
+  "version": "0.0.1",
```

Comment:
if this is the "production" version, consider using `v1.0.0`

Reply:
followed by the selector-parser versioning...not sure we want to commit to not breaking. 
WDTY?

Reply:
I'm not scared of major versions, but if you see it making breaking changes again several times in the near future then okay.